pytest_timeouts.py

```diff
@@ -13,6 +13,16 @@
 """
 
 
+@staticmethod
+def get_markers_old_way(item, name):
+    return item.get_marker(name=name)
+
+
+@staticmethod
```

Comment:
This weird syntax (staticmethod outside class) is required for python2 support

Reply:
okay then